Neos.ContentRepository/Tests/Unit/Domain/Service/NodeServiceTest.php

```diff
@@ -209,6 +210,8 @@ public function createChildNodesTryToCreateAllConfiguredChildNodes()
                 'second-child-node-name' => $secondChildNodeType
             ]));
 
+        $mockNode->method('getIdentifier')->willReturn(Uuid::uuid4());
```

Comment:
It's "only" a test, but what about `Neos\Flow\Utility::generateUUID()`? Yes, I know it uses `Uuid::uuid4()` as a fallback, but still… Shouldn't we use our own API?

Reply:
Good call